python/test/unit/io/test_adios2.py

```diff
@@ -234,3 +234,22 @@ def vel(x):
         f.write(t)
 
     f.close()
+
+
+@pytest.mark.skipif(not has_adios2, reason="Requires ADIOS2.")
```

Comment:
I guess this test should be marked with X-fail, as the output currently is not readable with Paraview?

Reply:
It won't produce output. The test is checking that an exception is raised.